tike/tomo.py

```diff
@@ -95,6 +95,7 @@
                         unicode_literals)
 
 import tomopy
+import tomopy.util.extern as extern
 import numpy as np
```

Comment:
Why do you call extern from TomoPy?

Reply:
I wanted to call `tompy.util.extern.c_project` directly because `tomopy.mproc` adds a lot of overhead, but I found that the results were inconsistent.